buku

```diff
@@ -3519,6 +3552,33 @@ def import_md(filepath: str, newtag: Optional[str]):
 
                     yield (url, title, delim_wrap(tags), None, 0, True, False)
 
+def import_rss(filepath: str, newtag: Optional[str]):
```

Comment:
You may want to [add at least one test](https://github.com/jarun/buku/blob/master/tests/test_buku.py#L550-L802) for this function.

Reply:
Done